src/main/java/com/direwolf20/buildinggadgets/common/tools/MathTool.java

```diff
@@ -0,0 +1,16 @@
+package com.direwolf20.buildinggadgets.common.tools;
+
+public final class MathTool {
```

Comment:
Is this intended for a future commit?

Reply:
it alreasy contains 2 Methods...

Reply:
No as in, the methods have no uses thus far.

Reply:
Oh, didn't notice that...

Reply:
Yes it is for future
It's a big mess in my workspace, so I only committed the abstraction part